_includes/header.liquid

```diff
@@ -5,7 +5,7 @@
     </h1>
 
     <p>
-      Date:
+      Event date:
       <time datetime="{{site.con_datetime | date: "%Y%m%dT%H:%M:%S%:z" }}P{{ site.con_duration }}H">
```

Comment:
Should we add timezone adjacent to this?

Reply:
Timezone inc. already.